universal-application-tool-0.0.1/app/services/question/ReadOnlyQuestionService.java

```diff
@@ -33,6 +33,8 @@
   /** Returns all scalars for this version. */
   ImmutableMap<Path, ScalarType> getAllScalars();
 
+  ActiveAndDraftQuestions getActiveAndDraftQuestions();
```

Comment:
Suggestion (blocking): please document this method

Reply:
Done.